pkg/queue/poller.go

```diff
@@ -86,9 +87,9 @@ func (p *Poller) Run(stopCh <-chan struct{}) {
 		select {
 		case <-ticker.C:
 			queues := p.queues.List()
+			threads := p.listThreads()
```

Comment:
why did we change this? can not the other thing work? need to work on this.

Reply:
Fixed in https://github.com/practo/k8s-worker-pod-autoscaler/pull/69/commits/81537618688edb07d3d0b0f33c031f6d896c598b